bin/generate_iframe_translations.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node 
+
+var languages = [
```

Comment:
This script is just something I wrote to automate the hack, but it should be cleaned up.